tests/lib/rules/await-async-utils.test.ts

```diff
@@ -103,18 +103,28 @@ ruleTester.run(RULE_NAME, rule, {
       `,
     })),
     ...ASYNC_UTILS.map((asyncUtil) => ({
+      settings: {
+        'testing-library/utils-module': 'test-utils',
+      },
       code: `
         import { ${asyncUtil} } from 'some-other-library';
-        test('util "${asyncUtil}" which is not related to testing library is valid', async () => {
+        test(
```

Comment:
After checking if coming from Testing Library directly inside `isAsyncUtil`, this test was invalid. I opted-out aggressive reporting so this test keeps passing, and moved the original one to invalid test cases.